flow/layers/child_scene_layer.h

```diff
@@ -27,16 +27,19 @@ class ChildSceneLayer : public Layer {
 
   void set_scene_token(uint32_t scene_token) { scene_token_ = scene_token; }
 
+  void set_prune(bool prune) { prune_ = prune; }
```

Comment:
I think we can do a better job of naming this method. It took me a while to make the connection that `set_prune` on the `ChildSceneLayer` was actually referring to the layers hit-test behavior (and only after opening up the `.fidl` file and the `.cc`). Maybe `set_hit_test_behavior` or something more indicative of this setter controlling hit-testing specifically?

Also, I am not sure I understand the relationship between `HitTestBehavior.Visibility.INVISIBLE` and `prune`.